src/ckb-daemon/usb.h

```diff
@@ -168,8 +172,14 @@ const char* product_str(short product);
 /// Used to apply quirks and features to the PLATINUM devices.
 #define IS_PLATINUM(kb)                 ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95_PLATINUM))
 
-/// Used when a device has a firmware with a low version number that uses the new protocol
-#define IS_V2_OVERRIDE(kb)             (IS_PLATINUM(kb) || IS_K63(kb) || IS_K68(kb) || IS_HARPOON(kb) || IS_GLAIVE(kb) || (kb)->product == P_STRAFE_NRGB_2)
+/// Used for even newer devices that come with V3 firmware out of the factory.
+#define IS_V3_OVERRIDE(kb)              (0)
+
+/// Used when a device has a firmware with a low version number that uses the new endpoint configuration.
```

Comment:
This comment might confuse people, because V3 has a *new* new endpoint configuration.